src/Donations/Components/PaymentsForm.tsx

```diff
@@ -26,7 +25,7 @@ import themeProperties from "../../../styles/themeProperties";
 import { ThemeContext } from "../../../styles/themeContext";
 import CheckBox from "../../Common/InputTypes/CheckBox";
 import { useRouter } from "next/router";
-import { CONTACT, PAYMENT, THANK_YOU } from "src/Utils/donationStepConstants";
+import { CONTACT, PAYMENT } from "src/Utils/donationStepConstants";
```

Comment:
type of `providerObject` argument of `onSubmitPayment` function is set to `any`
Please give it a valid type.